feature-configs/deploy/gatekeeper/gatekeeper-operator.yaml

```diff
@@ -0,0 +1,1058 @@
+apiVersion: v1
```

Comment:
@marzianor9 for better readability can you please split this file to
1. namespace
2. crd
3. deploy + service
4. role
5. clusterrole

Reply:
if we do this we will need to update separate parts when we have a new version right?